lib/gitscrub.js

```diff
@@ -80,36 +80,51 @@ gitscrub.selectRepos = function(reposToSelect, optionalFileName, logValue) {
       // first retrieve existing object from repos_to_scrub.json
       fs.readFile(path.join(__dirname, fileName), 'utf8', function(err, data) {
         if (err) {
-          throw err
+          logValue(err, null)
         }
         var reposToScrub = JSON.parse(data);
-        // first clear the stored list of repos
-        reposToScrub.repos = [];
         // check if repos specified in argument are valid and
         // push them onto our local reposToScrub object
         for (var repo in reposToSelect) {
-          if (repoList.indexOf(reposToSelect[repo]) > -1) {
+          if (repoList.indexOf(reposToSelect[repo]) > -1 && reposToScrub.repos.indexOf(reposToSelect[repo]) < 0) {
             reposToScrub.repos.push(reposToSelect[repo])
           }
         }
         // finally, write our modified repostoScrub object back to the file
         fs.writeFile(path.join(__dirname, fileName), JSON.stringify(reposToScrub, null, 2), function(err) {
           if (err) {
-            throw err
+            logValue(err, null)
           }
-          logValue(true, undefined)
+          logValue(null, true)
         })
       })
     })
   } else {
-    logValue(undefined, "Argument must be an array of repos to scrub")
+    logValue("Argument must be an array of repos to scrub", null)
   }
 }
 
+gitscrub.clearSelRepos = function(optionalFileName, logValue) {
```

Comment:
hm. Should we consider bunching this into gitscrub.reset? I already have it clearing the default fie